WordPress/Classes/Utility/Analytics/WPAppAnalytics.h

```diff
@@ -13,6 +13,7 @@ extern NSString * const WPAppAnalyticsKeyFeedID;
 extern NSString * const WPAppAnalyticsKeyFeedItemID;
 extern NSString * const WPAppAnalyticsKeyIsJetpack;
 extern NSString * const WPAppAnalyticsKeySessionCount;
+extern NSString * const WPAppAnalyticsKeySubscriptionCount;
```

Comment:
I guess we can move that to `WPAnalyticsEvent.swift`

In the future (ideally) we'd migrate all of that to Swift.